doc/source/examples/index.rst

```diff
@@ -18,17 +18,7 @@ Here are a series of examples using MAPDL with ``ansys-mapdl-core``.
 
 .. === TECHNOLOGY SHOWCASE EXAMPLES ===
 
-.. toctree::
-    :hidden:
-    :maxdepth: 3
-
-    technology_showcase_examples/techdemo-1/ex_0-tecbrakesqueal
-    technology_showcase_examples/techdemo-20/20-example-technology-showcase-dynamic-simulation-PCB
-    technology_showcase_examples/techdemo-28/ex_28-tecfricstir
-
-.. include:: technology_showcase_examples/index.rst
-   :start-line: 2
-
+We will add the link the pymapdl-techdemos repository.
```

Comment:
Improve a little bit the description here @clatapie:
* What are the techdemos? Explain that these examples are hosted in a different repository
* Where can they be found?
* Provide links to the documentation and to the repository

Reply:
I will make the pymapdl-techdemos repository public, no worries